docs/content/guides/author-apps/kubernetes/how-to-patch-pod/index.md

```diff
@@ -0,0 +1,133 @@
+
+   ```bash
+   kubectl get pods -n <YOUR_KUBERNETES_NAMESPACE> -l app.kubernetes.io/name=demo -o custom-columns=POD:.metadata.name,STATUS:.status.phase,CONTAINER_NAMES:spec.containers[:].name,CONTAINER_IMAGES:spec.containers[:].image
+   ```
+
+   You should see output confirming that a single container named `demo` was deployed and is running in your pod, similar to the following:
+
+   ```
+   POD                    STATUS    CONTAINER_NAMES   CONTAINER_IMAGES
+   demo-df76d886c-9p4gv   Running   demo              radius.azurecr.io/tutorial/webapp:edge
+   ```
+
+## Step 3: Add a PodSpec to the container definition
+
+Add the following [`runtimes`]({{< ref "reference/resource-schema/core-schema/container-schema#runtimes" >}}) under `properties` of the container definition in your `app.bicep` file:
+
+{{< rad file="snippets/patch-runtime.bicep" embed=true markdownConfig="{linenos=table,hl_lines=[\"16-28\"]}" >}}
+
+> Remember to save your `app.bicep` file after you've made the above changes.
+
+## Step 4: Redeploy your app with the PodSpec added
+
+1. Deploy and run your app again:
+
+   ```bash
+   rad run ./app.bicep
+   ```
+
+   Once the deployment completes successfully, you should see the same deployment completion confirmation message as before, but this time with some system logs from `log-collector` streaming to your console output:
+
+   ```
+   Starting log stream...
+
+   + demo-547d7dc77f-nmqpk › log-collector
+   + demo-547d7dc77f-nmqpk › demo
+   demo-547d7dc77f-nmqpk log-collector Fluent Bit v2.1.8
+   demo-547d7dc77f-nmqpk log-collector * Copyright (C) 2015-2022 The Fluent Bit Authors
+   demo-547d7dc77f-nmqpk log-collector * Fluent Bit is a CNCF sub-project under the umbrella of Fluentd
+   demo-547d7dc77f-nmqpk log-collector * https://fluentbit.io
+   demo-547d7dc77f-nmqpk log-collector
+   ```
+
+   Access the application by opening [http://localhost:3000](http://localhost:3000) in a browser, where you should see the demo app again unchanged from before:
+
+   <img src="./demoapp-howtopatchpod.png" alt="Screenshot of Radius Demo app" width="600"/>
+
+   <br> When you're ready to move on to the next step, use `CTRL` + `C` to exit the command.
+
+2. Run the following command, making sure to specify the Kubernetes namespace to which you deployed your app:
+
+   ```bash
+   kubectl get pods -n <YOUR_KUBERNETES_NAMESPACE> -l app.kubernetes.io/name=demo -o custom-columns=POD:.metadata.name,STATUS:.status.phase,CONTAINER_NAMES:spec.containers[:].name,CONTAINER_IMAGES:spec.containers[:].image
+   ```
+
+   You should now see in the output the original `demo` app container as before, but also an additional `log-collector` container that is running in your pod, similar to the following:
+
+   ```
+   POD                     STATUS    CONTAINER_NAMES      CONTAINER_IMAGES
+   demo-547d7dc77f-nmqpk   Running   log-collector,demo   ghcr.io/radius-project/fluent-bit:2.1.8,radius.azurecr.io/tutorial/webapp:edge
+   ```
```

Comment:
This is a really cool way of demonstrating the effect. 

Depending on the speed the user does this, they might see old pods with a state of `Terminating`. You might want to mention that, and say that it's normal.